modular_zubbers/code/modules/mob/living/carbon/human/species/xeno.dm

```diff
@@ -1,3 +1,43 @@
 /datum/species/xeno
 	mutant_organs = list()
 	heatmod = 1
+	inherent_traits = list(
+		TRAIT_ADVANCEDTOOLUSER,
+		TRAIT_CAN_STRIP,
+		TRAIT_LITERATE,
+		TRAIT_MUTANT_COLORS,
+		TRAIT_NIGHT_VISION
+	)
+	species_language_holder = /datum/language_holder/xeno_hybrid
+
+/datum/species/xeno/get_species_description()
+	return list(
+		"A 'Xenomorph Hybrid' is a loose designation for a wide variety of creatures with predominantly \
+		Xenomorph-like characteristics, rather than a specific species in its own right. As Xenomorphs themselves are \
+		parasitic hybridisations of many different 'host' species, the main distinguishing factors between a regular \
+		Xenomorph and a Hybrid are a more humanoid body shape and stature with alien features, inert or defective hive \
+		organs and a less murderous demeanour. These are not the only documented differences, though they are the most frequent.",
+
+		"Their manner of creation just as wildly varies, from lab-grown experiments to unusual genetic malformations \
+		in pure Xenomorph hives. All of this unfortunately culminates into a degree of prejudice and fear from the other \
+		races for their roots as parasites, regardless of if the Hybrid in question is even capable of performing such infestuous acts.",
```

Comment:
```suggestion
		in pure Xenomorph hives. All of this unfortunately culminates into a degree of prejudice and fear from  other \
		races for their origins as parasites, regardless of if the Hybrid in question is even capable of infesting other living creatures.",
```

Reply:
`infesting [...] creatures` is more a suggestion than a requirement to change - up to you.